jenkins/ci.suse.de/cloud-mkcloud8.yaml

```diff
@@ -0,0 +1,102 @@
+- project:
+    name: cloud-mkcloud8-gating
+    version: 8
+    previous_version: 7
+    label: cloud-trigger
+    jobs:
+        - 'cloud-mkcloud{version}-gating'
+- project:
+    name: cloud-mkcloud8-x86_64
+    disabled: false
+    version: 8
+    previous_version: 7
+    arch: x86_64
+    tempestoptions: --smoke
+    label: openstack-mkcloud-SLE12-x86_64
+    jobs:
+        - 'cloud-mkcloud{version}-job-2nodes-{arch}'
+        - 'cloud-mkcloud{version}-job-4nodes-linuxbridge-{arch}'
+          # - 'cloud-mkcloud{version}-job-backup-restore-{arch}'
+          # - 'cloud-mkcloud{version}-job-btrfs-{arch}'
+          #- 'cloud-mkcloud{version}-job-crowbar_register-{arch}'
+          #- 'cloud-mkcloud{version}-job-crowbar-devsetup-{arch}'
+          #- 'cloud-mkcloud{version}-job-dvr-{arch}'
+          #- 'cloud-mkcloud{version}-job-magnum-{arch}'
+          #- 'cloud-mkcloud{version}-job-monasca-{arch}'
+          #- 'cloud-mkcloud{version}-job-raid-{arch}'
+          #- 'cloud-mkcloud{version}-job-ssl-{arch}'
+          #- 'cloud-mkcloud{version}-job-upgrade-disruptive-{arch}'
+          #- 'cloud-mkcloud{version}-job-xen-{arch}'
```

Comment:
Do we need them commented? It's saved elsewhere, so let's rather keep this file clean...